configure.py

```diff
@@ -1100,14 +1100,17 @@ def configure_node(o):
   o['variables']['node_shared'] = b(options.shared)
   node_module_version = getmoduleversion.get_version()
 
-  if sys.platform == 'darwin':
-    shlib_suffix = '%s.dylib'
-  elif sys.platform.startswith('aix'):
-    shlib_suffix = '%s.a'
+  if options.dest_os == 'android':
+    shlib_suffix = 'so'
   else:
-    shlib_suffix = 'so.%s'
+    if sys.platform == 'darwin':
+      shlib_suffix = '%s.dylib'
+    elif sys.platform.startswith('aix'):
+      shlib_suffix = '%s.a'
+    else:
+      shlib_suffix = 'so.%s'
+    shlib_suffix %= node_module_version
```

Comment:
Just for my understanding... You want to avoid the __shlib_suffix %= node_module_version__ step on android?

Reply:
You could minimize the diff by making the string interpolation conditional:
```python
if '%s' in shlib_suffix:
  shlib_suffix %= node_module_version
```

Reply:
@bnoordhuis This is resolved to your satisfaction?

Reply:
Yes. I'll hit the 'Resolve conversation' button.